packets.go

```diff
@@ -632,6 +632,16 @@ func (rows *textRows) readRow(dest []driver.Value) error {
 							string(dest[i].([]byte)),
 							mc.cfg.loc,
 						)
+
+						if err != nil && mc.invalidTimeAsZero {
+							//in some mysql configurations - possible to store dates in not valid format,
```

Comment:
```go
// Some MySQL configurations allow the storage of values in an invalid date format.
// With this option they are interpreted as zero-dates instead of returning an error.